types/scte224v20200407/sctestructures_test.go

```diff
@@ -0,0 +1,73 @@
+package scte224v20200407
```

Comment:
Added tests only for the new additions. Tests are not exhaustive, but whether it has to be is a question. Because, we don't expect too many changes here, once they are locked in